front/app/services/appConfiguration.ts

```diff
@@ -78,6 +78,7 @@ export interface IAppConfigurationSettings {
   password_login?: {
     allowed: boolean;
     enabled: boolean;
+    enable_signup: boolean;
```

Comment:
This comment is mainly for my own learning, I see in the schema that this is `default: true` on the backend, does that mean it will be auto-populated for existing customers?

Reply:
Exactly.